runtime/runtime-params-estimator/estimator-warehouse/src/check.rs

```diff
@@ -0,0 +1,135 @@
+use std::collections::BTreeSet;
+
+use clap::Parser;
+
+use crate::{
+    db::{EstimationRow, DB},
+    zulip::{ZulipEndpoint, ZulipReport},
+    Metric,
+};
+
+#[derive(Parser, Debug)]
+pub(crate) struct CheckConfig {
+    /// Send notifications from checks to specified server.
+    #[clap(long, default_value = "near.zulipchat.com")]
+    zulip_server: String,
```

Comment:
Do we need this to be a CLI option?

Reply:
Probably not. I will remove it. (I was thinking about development with a local Zulip instance. But then I discovered I can easily send to my private user instead of the stream.)